config/initializers/logging.rb

```diff
@@ -0,0 +1,10 @@
+# Logging configuration for the whole app.
+#
+# For details see https://github.com/TwP/logging
+
+require 'logging'
+
+Logging.logger.root.appenders = Logging.appenders.stdout(
+  layout: Logging::Layouts.pattern(format_as: :inspect, pattern: '%-5l [%c]: %m\n')
+)
+Logging.logger.root.level = (ENV['RACK_ENV'] == 'production' ? :info : :debug)
```

Comment:
I _suppose_ this could use `Config.rack_env` so there's a guaranteed default, but this is also fine.